### PR TITLE
Add link list

### PR DESCRIPTION
**Pull request by jnmarcus**

### CODE REVIEW

**Comment by KyleW** [comment]
Looks good to me.

**Comment by jnmarcus** [comment]
k, the tests seem to be having various issues, one of them being that it can't find jQuery.  Since I plan to take jQuery out of the tests (see #115  and #96 for some more info), I have a feeling some of the reasons the tests are breaking may resolve themselves so I think I'm going to merge this for now.

**Comment by jnmarcus** [comment]
Here is the screenshot to go along with this component:
screen shot 2016-07-12 at 8 11 06 pm

**Comment by jnmarcus** [comment]
as a heads up, I've been going back and forth about the best way to build this.  I'm still not positive about my decision, but I wanted to get v1 out at least, so we'll take it from there.  Feedback is welcome tho, and we can always review if needed.  Feel free to file issues 😄

**Comment by jnmarcus** [comment]
related to #91

File: app/components/common/linkList/linkListDirective.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    };
+    return directive;
+
+    function link(scope, element, attrs, vm) {
+      // var listTitle = attrs['listTitle'] || null;
+      var childElements = element.children();
+
+      if (!vm.listTitle || vm.listTitle === null || undefined) {
+        childElements.find('h3').remove();
+      }
```
Comment (KyleW) [inline app/components/common/linkList/linkListDirective.js]: This parts a bit funky. It's probably easier to just throw an ng-if="vm.listTitle" on your h3 element in the html.

Comment (jnmarcus) [inline app/components/common/linkList/linkListDirective.js]: @KyleW definitely.  I'll fix it now.

**Pull request merged**